### DISCUSSION TITLE
When sharing an image link, add the post title with it

### DISCUSSION THREAD

**Discussion by SinTan1729** [Docile-Alligator/Infinity-For-Reddit#966] [Ideas]
Sync does this. Whenever you share the image URL, you can set it up so that it also shares the post title with it. It looks like this:
\<Post title>
\<Image link>
(In different lines)

**Comment by AB1908** [comment]
I'd like something in this vein as well, specifically to use with Orgzly. Similar issue opened here: https://github.com/Mosc/Glider/issues/43

**Comment by rheicide** [comment]
Would be nice to have this feature. It could be very useful when using Share Post Link to send a post to a todo or bookmark app and have the title populated automatically.
